rosbag2_py/package.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>rosbag2_py</name>
+  <version>0.2.4</version>
+  <description>Python API for rosbag2</description>
+  <maintainer email="jacob@openrobotics.org">Jacob Perron</maintainer>
+  <maintainer email="ros-tooling@googlegroups.com">ROS Tooling Working Group</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake_ros</buildtool_depend>
+
+  <depend>pybind11_vendor</depend>
+  <depend>rosbag2_cpp</depend>
+  <depend>rosbag2_compression</depend>
+
+  <test_depend>ament_copyright</test_depend>
+  <test_depend>ament_flake8</test_depend>
+  <test_depend>ament_pep257</test_depend>
```

Comment:
I think we should add some C++ linters. Maybe just replace all of these with the common linters?
We're also missing a test depend on `ament_lint_auto`.

```suggestion
  <test_depend>ament_lint_auto</test_depend>
  <test_depend>ament_lint_common</test_depend>
```

Reply:
Niiiiice I got like a hundred errors after that. I was wondering why nothing complained...
d261c63